test/1_stdlib/ArrayCore.swift

```diff
@@ -13,44 +13,9 @@
 // REQUIRES: executable_test
 
 import Swift
+import StdlibUnittest
 
-//===--- class Tracked ----------------------------------------------------===//
-// Instead of testing with Int elements, we use this wrapper class
-// that can help us track allocations and find issues with object
-// lifetime inside Array implementations.
-var trackedCount = 0
-var nextTrackedSerialNumber = 0
-
-final class Tracked : ForwardIndex, CustomStringConvertible {
-  required init(_ value: Int) {
-    trackedCount += 1
-    nextTrackedSerialNumber += 1
-    serialNumber = nextTrackedSerialNumber
-    self.value = value
-  }
-  
-  deinit {
-    assert(serialNumber > 0, "double destruction!")
-    trackedCount -= 1
-    serialNumber = -serialNumber
-  }
-
-  var description: String {
-    assert(serialNumber > 0, "dead Tracked!")
-    return value.description
-  }
-
-  func successor() -> Self {
-    return self.dynamicType.init(self.value.successor())
-  }
-
-  var value: Int
-  var serialNumber: Int
-}
-
-func == (x: Tracked, y: Tracked) -> Bool {
-  return x.value == y.value
-}
+typealias Tracked = LifetimeTracked
```

Comment:
Feel free to rename things across the test.  Having leftover migration aids does not help code maintainability in future.